rgw/admin/user_test.go

```diff
@@ -104,6 +105,8 @@ func (suite *RadosGWTestSuite) TestUser() {
 		user, err := co.GetUser(context.Background(), User{ID: "leseb"})
 		assert.NoError(suite.T(), err)
 		assert.Equal(suite.T(), "leseb@example.com", user.Email)
+		assert.Equal(suite.T(), "users", user.Caps[0].Type)
```

Comment:
Test mostly looks OK, but how would I know this isn't the default value?

Reply:
the default value is nil for `caps`. Do want you add check whether `Caps` is not nil here?

Reply:
if caps was nil by deafult caps[0] will panic failing the test. so this seems OK to me in light of that.